BusBoard.Web/Views/Home/BusInfo.cshtml

```diff
@@ -1,10 +1,72 @@
-﻿@model BusBoard.Web.ViewModels.BusInfo
+﻿@using Microsoft.AspNetCore.Html
+@using BusBoard.Api.Tfl
+@using BusBoard.Api.Tfl.Responses
+@using BusBoard.Web.ViewModels
+@model BusBoard.Web.ViewModels.BusInfo
 @{
-  ViewBag.Title = "Bus Information";
+    ViewBag.Title = "Bus Information";
+}
+
+@functions {
+
+  static string FormatMinutes(int minutes) => minutes == 1 ? "1 min" : $"{minutes} mins";
+
+}
+
+@helper RenderPrediction(Prediction prediction)
```

Comment:
Using functions in a cshtml function isn't very common. I'd just define this inline, I think it's more readable that way, you can more easily see the structure. If you think the file is getting too long you can use a partial: https://docs.microsoft.com/en-us/aspnet/core/mvc/views/partial?view=aspnetcore-3.1 🙂

Reply:
I'll have a look into partials but I agree that inlining it results in completely readable code. Changed in 418a34d 🙂